### DISCUSSION TITLE
What do you know about Rest API

### DISCUSSION THREAD

**Discussion by omphilejmatsobe** [community/community#56870] [Apps, API and Webhooks]
Is it a must for every FrontEnd and Backend developer to know Rest API and how to use it ?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @omphilejmatsobe, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
